include/SisterRay/data_addresses.h

```diff
@@ -5,4 +5,51 @@
 #define ARMOR_DATA_PTR (void*)0x00dbcce0
 #define ACCESSORY_DATA_PTR (void*)0x00dbcae0
 
+#define SAVEMAP_INVENTORY_PTR = (void*)0x00DC0234
```

Comment:
Stray `=` (Probably from an earlier global)

```suggestion
#define SAVEMAP_INVENTORY_PTR (void*)0x00DC0234
```